earth_enterprise/src/fusion/autoingest/StorageManager.h

```diff
@@ -355,5 +368,30 @@ bool StorageManager<AssetType>::SaveDirtyToDotNew(
   return true;
 }
 
+template<class AssetType>
+bool StorageManager<AssetType>::DetermineIfPrune() {
+  std::lock_guard<std::recursive_mutex> lock(storageMutex);
+  uint32 cacheSize = CacheSize();
+  uint32 dirtySize = DirtySize();
+  uint32 oriNum;
+  uint32 newNum;
+  uint32 diff;
+  float percent;
+
+  if (cacheSize > dirtySize) {
```

Comment:
`cacheSize` will always be >= `dirtySize` - if it's in the dirty map, it's also in the cache.